comb-browse.el

```diff
@@ -58,7 +58,7 @@
     ("q" . comb-quit))
   "Default keybindings.")
 
-(defconst comb-keymap
+(defvar comb-keymap
```

Comment:
Why is that?

Reply:
By convention, keymaps seem to be defined with defvar. I guess it makes sense, since doing `(setq comb-keymap ..)` should be allowed.

Reply:
You still can:

> The difference between defconst and defvar is mainly a matter of intent, serving to inform human readers of whether the value should ever change. Emacs Lisp does not actually prevent you from changing the value of a variable defined with defconst.

Reply:
Indeed. I was mostly following convention found in other packages. Feel free to revert if you prefer defconst.